src/pykeen/datasets/openea.py

```diff
@@ -0,0 +1,185 @@
+        :param side:
+            The side of the graph-pair, a substring of the graph-pair selection.
+        :param size:
+            The size of the graphs (either "15K" or "100K").
+        :param version:
+            The version of the pairing (either "V1" or "V2). "V1" has lower connectivity in the graph than "V2".
+        :param cache_root:
+            The cache root.
+        :param eager:
+            Whether to directly load the dataset, or defer it to the first access of a relevant attribute.
+        :param create_inverse_triples:
+            Whether to create inverse triples.
+        :param random_state:
+            The random state used for splitting.
+        :param split_ratios:
+            The split ratios used for splitting the dataset into train / validation / test.
+        :param force:
+            Whether to enforce re-download of existing files.
+
+        :raises ValueError:
+            If the graph pair or side is invalid.
+        """
+        # Input validation.
+        if graph_pair not in GRAPH_PAIRS:
+            raise ValueError(f"Invalid graph pair: Allowed are: {GRAPH_PAIRS}")
+        available_sides = graph_pair.split("_")
+        if side not in available_sides:
+            raise ValueError(f"side must be one of {available_sides}")
+        if size not in GRAPH_SIZES:
+            raise ValueError(f"size must be one of {GRAPH_SIZES}")
+        if version not in GRAPH_VERSIONS:
+            raise ValueError(f"version must be one of {GRAPH_VERSIONS}")
+
+        relative_path_base = pathlib.PurePosixPath(
+            "OpenEA_dataset_v2.0",
+            graph_pair + "_" + size + "_" + version,
+        )
+        # left side has files ending with 1, right side with 2
+        one_or_two = "1" if side == available_sides[0] else "2"
+        self._relative_path_attributes = pathlib.PurePosixPath(relative_path_base, f"attr_triples_{one_or_two}")
+        self._relative_path_relations = pathlib.PurePosixPath(relative_path_base, f"rel_triples_{one_or_two}")
+
+        # For downloading
+        self.force = force
+        self.cache_root = self._help_cache(cache_root)
+
+        # For splitting
+        self.random_state = random_state
+        self.ratios = split_ratios
+
+        # Whether to create inverse triples
+        self.create_inverse_triples = create_inverse_triples
+
+        if eager:
+            self._load()
+
+    def _load(self) -> None:
+        path = self.cache_root.joinpath("OpenEA_dataset_v2.0.zip")
+
+        # ensure file is present
+        if not path.is_file() or self.force:
+            logger.info(f"Downloading file from Dropbox (Link: {self.__class__.DROPBOX_LINK})")
+            download(url=self.__class__.DROPBOX_LINK, path=path, hexdigests={"sha512": self.SHA512})
+
+        read_csv_kwargs = {
+            "header": None,
+            "names": [LABEL_HEAD, LABEL_RELATION, LABEL_TAIL],
+            "sep": "\t",
+            "engine": "python",
```

Comment:
if it's possible to use the C engine, then it's preferred. maybe you should explicitly set the `compression="gzip"` if this was giving you an issue before

Reply:
additionally, you can use `"dtype":str"` so you don't have to recast later (probably speeds parsing up a bit too)

Reply:
I use the default engine now. Using "dtype":str did not work unfortunately due to parsing errors. So I'll stick with the recasting.